soak/README.md

```diff
@@ -1,317 +1,117 @@
-    ```
-            
-* Above command will install the helm chart which will run your soak tests. You can now view live metrics on Grafana console
-or check the test results later when soak tests are finished after "$NET_SOAK_TEST_DURATION_MINUTES" minutes.
-
-NOTE:
-* Currently soak-test job uses the same service account for running soak tests that is running the service controller.
-* E2E tests that run as part of soak tests require aws credentials for test setup and validation steps of those tests.
-
-Validation:
-* After executing above command, a Kubernetes Job will start which will execute the soak tests until complete. 
-* Run `kubectl get job/$SERVICE_NAME-soak-test`  and validate that the resource exists.
-
-### Step 6 (Monitor the metrics using Grafana console)
-
-* As mentioned in step 2, Prometheus and Grafana services are exposed through background port-forward processes.
-* You can access the Grafana dashboard in your browser at `http://127.0.0.1:$LOCAL_GRAFANA_PORT` address. NOTE: Do not forget
-to substitute "$LOCAL_GRAFANA_PORT" with actual value from your `~/.ack/soak/config` file
-* When Grafana console is loaded, you can login with default username `admin` and default password `prom-operator`
-NOTE: You can see the value of these credentials in secret named `$PROM_CHART_RELEASE_NAME-grafana`
-* After the login, you can view your Grafana dashboard at `http://127.0.0.1:3000/dashboards`.
-* The `ACK Dashboard` will show the ACK-specific request counts and error codes when making calls to AWS APIs.
-* `Kubernetes/Compute Resources/Pod` dashboard will show the resource consumption by the controller pod
-
-
-### Step 7 (Cleanup Soak test chart)
-
-* RUN `helm -n ack-system uninstall $SOAK_RUNNER_CHART_RELEASE_NAME`
-
-### Step 8 (Cleanup the service controller deployment)
-
-* RUN `helm -n ack-system uninstall $CONTROLLER_CHART_RELEASE_NAME && kubectl delete namespace ack-system`
-
-### Step 9 (Optional: Cleanup the kube-prometheus-stack chart) 
-
-* RUN `helm -n prometheus uninstall $PROM_CHART_RELEASE_NAME && kubectl delete namespace prometheus`
-
-### Step 10 (Cleanup the background port-forward processes)
-
-* RUN 
-    ```bash
-    for pid in $(ps -a | grep "kubectl port-forward" | grep "$PROM_CHART_RELEASE_NAME" | cut -d" " -f1); do
-        kill -9 $pid ;
-    done
-    ```
\ No newline at end of file
+Create a new AWS account to house the following resources. A new account ensures
+that any leaked resources do not affect the e2e tests running on your pull
+requests, and that you can give the ACK core team permissions to view this
+account.
+
+Export the credentials for an AWS user with administrator access to your
+terminal before following the rest of the documents.
+
+### Step 1 (Run the bootstrapping script)
+
+Run the `bootstrap.sh` script in the current directory. This script will do the
+following: 
+1. Create an ECR public repository for the soak test runner
```

Comment:
the ordered list only has "1" number

Reply:
With markdown you can create an ordered list by repeating `1.` and the renderer should fix the sequencing for you. At least Github does.